src/.well-known/mta-sts.txt

```diff
@@ -0,0 +1,6 @@
+version: STSv1
+mode: enforce
```

Comment:
can we do `mode: testing` and check it with an MTA-STSv1 tester before we go live?

Reply:
(Fully understanding we don't have `TLSRPT` and won't get the reports)

Reply:
We do have TLSRPT; I already set that up.